docs/network/ble/bletiny/bletiny_api.md

```diff
@@ -1,6 +1,8 @@
 ## API for bletiny app
 
-"bletiny" is one of the sample applications that come with Mynewt. It is a simple shell application which provides a basic interface to the host-side of the BLE stack. "bletiny" includes all the possible roles (Central/Peripheral) and they may be run simultaneously. You can run bletiny on a board and issue commands that make it behave as a central or a peripheral with different peers. 
+"bletiny" is one of the sample applications that come with Mynewt. It is a shell application which provides a basic interface to the host-side of the BLE stack. "bletiny" includes all the possible roles (Central/Peripheral) and they may be run simultaneously. You can run bletiny on a board and issue commands that make it behave as a central or a peripheral with different peers. 
+
+"bletiny" is a legacy BLE application. If you want to use new BLE application please go to [btshell app](../btshell/btshell_api.md).
```

Comment:
Aside from using the 1.0 shell, what else is different in terms of "legacy". Does bletiny use BLE 4.2 and btshell uses  some BLE 5.0 features?